LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2012, Met Office.
```

Comment:
This is a new license, so does the copyright year also need to be updated in addition to @rcomer's suggestion of Cartopy Contributors instead of Met Office?
```suggestion
Copyright (c) 2023, Cartopy Contributors
```

Or optionally, can you put multiple copyrights into the file, but this may imply that this license was valid for previous dates?
`Copyright (c) 2012-2023, Met Office.`